core/trino-main/src/test/java/io/trino/sql/query/QueryAssertions.java

```diff
@@ -501,6 +502,34 @@ public QueryAssert isFullyPushedDown()
             return this;
         }
 
+        /**
+         * Verifies query is fully pushed down and Table Scan is replaced with empty Values.
+         * Verifies that results are the same as when pushdown is fully disabled.
+         */
+        @CanIgnoreReturnValue
+        public QueryAssert isReplacedWithEmptyValues()
+        {
+            checkState(!(runner instanceof LocalQueryRunner), "isReplacedWithEmptyValues() currently does not work with LocalQueryRunner");
```

Comment:
add coverage in `BaseQueryAssertionsTest`, see `io.trino.tests.BaseQueryAssertionsTest#testIsFullyPushedDown`